src/App.tsx

```diff
@@ -45,10 +47,10 @@ function App() {
         <Route path="/dashboard">
           <Dashboard />
         </Route>
-        <Route path="/changePassword">
+        <Route path="/change-password">
```

Comment:
This breaks the navigation component. We can discuss changing the routing naming convention but not the night before the end of the sprint.